src/concrete/ml/pandas/_utils.py

```diff
@@ -133,28 +133,28 @@ def deserialize_elementwise(array: numpy.ndarray) -> numpy.ndarray:
     return numpy.vectorize(deserialize_value)(array)
 
 
-def serialize_evaluation_keys(evaluation_keys: fhe.EvaluationKeys) -> str:
+def serialize_evaluation_keys(evaluation_keys: fhe.EvaluationKeys) -> bytes:
```

Comment:
why do you need hex? it requires processing that is slow

Reply:
I removed it actually, just forgot to update the docstring as well good catch

initially it was for making it serializable in json